notifier/handlers_test.go

```
@@ -145,6 +145,66 @@ func TestHandlers(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:       "Get Concepts - Success",
+			method:     "GET",
+			url:        fmt.Sprintf("/concepts?lastChangeDate=%s", today),
+			resultCode: 200,
+			resultBody: `["1","2","3"]`,
+			mockService: &mockService{
+				getConcepts: func(t time.Time) ([]string, error) {
+					return []string{"1", "2", "3"}, nil
+				},
+			},
+		},
+		{
+			name:       "Get Concepts - Invalid Time",
+			method:     "GET",
+			url:        fmt.Sprintf("/concepts?lastChangeDate=%s", past),
+			resultCode: 400,
+			resultBody: fmt.Sprintf("{\"message\": \"Last change date should be time point in the last %.0f hours\"}", LastChangeLimit.Hours()),
+			mockService: &mockService{
```

Since we return 400, I guess the mockservice should also return an error. Same applies to the next 3 tests.